pages/Receive.qml

```diff
@@ -362,6 +462,7 @@ Rectangle {
         if (appWindow.currentWallet) {
             appWindow.current_address = appWindow.currentWallet.address(appWindow.currentWallet.currentSubaddressAccount, 0)
             appWindow.currentWallet.subaddress.refresh(appWindow.currentWallet.currentSubaddressAccount)
+            subaddressListView.currentIndex = 0
```

Comment:
Any reason for always resetting to index 0?

Reply:
when the pages open for the first time, subaddressListView.currentIndex is undefined (= -1). 
Now it will only change to zero when it's undefined.